.circleci/config.yml

```diff
@@ -829,6 +829,11 @@ jobs:
           command: |
             # lsof is used by Colony in its stop-blockchain-client.sh script
             sudo apt-get -qy install lsof
+            # create fake vyper
```

Comment:
You shouldbe root and could also write to /usr/bin. On the other hand, it would be nice to also make this work without the CI setup steps - but let's first fix this in some way.

Reply:
Ah yes, thats true. I will now just check whether this fixes the problem. If yes, I will move the "code" into `test/externalTests/common.sh` into the `setup` function.